src/test_utils.rs

```diff
@@ -1,20 +1,26 @@
 #![cfg(feature = "alloc")]
 use std::prelude::v1::*;
 
-use crate::types::CertificateDer;
-
 /// Signature algorithm used by certificates and parameters generated using the test utils helpers.
 static RCGEN_SIGNATURE_ALG: &rcgen::SignatureAlgorithm = &rcgen::PKCS_ECDSA_P256_SHA256;
 
-pub(crate) fn make_issuer(org_name: impl Into<String>) -> rcgen::Certificate {
-    rcgen::Certificate::from_params(issuer_params(org_name)).unwrap()
+pub(crate) fn make_issuer(org_name: impl Into<String>) -> rcgen::CertifiedKey {
```

Comment:
Decent time to move the helper(s) below the tests?

Reply:
There aren't any tests in this file (it's all crate internal helpers) but I think the members need a reorder in any case